README.md

```diff
@@ -35,7 +35,6 @@ You can watch videos of the system being developed on YouTube:
 * Pseudoterminals (with /dev/pts filesystem)
 * Filesystem notifications
 * CPU and memory profiling
-* SoundBlaster 16 driver
```

Comment:
How about replacing this with "AC97 audio driver"?

Reply:
That can be done separately :)

Reply:
If you feel like it is a good addition, go for it, but I don't really think we should add that. We have plenty of kernel features and hardware support to advertise but we don't do that, so I'm not sure how is this different in this case too.

Reply:
Also, if we go into full potential of "advertisement", I'd wait until we have Intel HDA support because that seems much more interesting than AC97 :)
Maybe we have to do that differently though, so we can have a separate list of what is supported in the Kernel. Probably not worth it right now, maybe in the future.